piicatcher/__init__.py

```diff
@@ -59,6 +59,18 @@ class SSN(PiiType):
     pass
 
 
+class ZipCode(PiiType, type="zip_code"):  # type: ignore
```

Comment:
What's the purpose of the `type` parameter? Doesn't look like it's getting used.

Reply:
PiiType class requires a type to be detected. Without specifying the type, when dbcat tries to `data.get("type")`, typer will return None for ZipCode.